api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1218,12 +1218,11 @@ ml_check_nnfw_availability (ml_nnfw_type_e nnfw, ml_nnfw_hw_e hw,
 
   if (fw_name) {
     if ((fw = nnstreamer_filter_find (fw_name)) != NULL) {
-      if (fw->checkAvailability
-          && fw->checkAvailability (ml_nnfw_to_accl_hw (hw)) != 0) {
+      *available = gst_tensor_filter_check_hw_availability
+        (fw, ml_nnfw_to_accl_hw (hw));
+      if (*available == false) {
```

Comment:
Q. Would there be any difference between `false` and `FALSE` as `tensor_filter_common.c:1946` is using `FALSE`?

Reply:
`FALSE` is a macro by glib which equals to `0`. So, I don't think there would be any difference.
As `available` is a bool (and not `gboolean` type), `false` is used instead of `FALSE`.

Reply:
Thanks for the explanation :)